### PR TITLE
Updates - theme, grammar, z-position

### PR DESCRIPTION
**Pull request by judyfong**
* Update the theme to be consist across the whole website.
* Fix the grammar on the front page
* Display webrice over every other element on a webpage.

### CODE REVIEW

File: build/maltaekni.html [hunk 1]
```diff
@@ -44,10 +45,11 @@ <h2 id="content-9">Skipulag námsins í HR</h2>
             <p id="content-10">Um er að ræða tveggja ára, þverfaglegt, 120 eininga nám. Einingarnar skiptast í 44-78 einingar úr námskeiðum á meistarastigi (kennd í HR og HÍ), 0-30 einingar úr grunnnámskeiðum í tölvunarfræði, 0-10 einingar úr grunnnámskeiðum í málfræði (kennd í HÍ) og 30-60 einingar í meistaraprófsverkefni/ritgerð. Samsetning eininga er því  sveigjanleg og fer eftir bakgrunni viðkomandi nemanda.</p>
             <p id="content-11">Nemendur með BA-próf í málvísindum og tungumálum þurfa að taka 30 einingar í grunnnámskeiðum í tölvunarfræði. Þessi námskeið eru metin sem hluti meistaranámsins. Nemendur með aðra undirstöðu gætu þurft að taka grunnnámskeið í bæði málfræði og tölvunarfræði.</p>
         </div>
-        <footer class="footer mt-auto bg-dark">
+      </div>
+        <footer class="footer mt-auto bg-grains">
```
Comment (rkjaran) [inline build/maltaekni.html:49]: Weird indentation here

File: build/solskin.html [hunk 2]
```diff
@@ -43,10 +44,11 @@ <h1 id="content-1">Sólskin</h1>
                 </div>
             </div>
         </div>
-        <footer class="footer mt-auto bg-dark">
-            <nav class="navbar">
+      </div>
+        <footer class="footer mt-auto">
+            <nav class="navbar bg-grains">
```
Comment (rkjaran) [inline build/solskin.html:49]: Weird indentation

File: build/maltaekni.html (hunk 1 above)
Comment (judyfong) [reply build/maltaekni.html]: fixed

File: build/solskin.html (hunk 2 above)
Comment (judyfong) [reply build/solskin.html]: fixed

**Pull request merged**